src/server/service/kv.rs

```diff
@@ -915,6 +833,7 @@ impl<T: RaftStoreRouter + 'static, E: Engine> tikvpb_grpc::Tikv for Service<T, E
 
         let task = EndPointTask::Request(req_task);
         if let Err(e) = self.end_point_scheduler.schedule(task) {
+            // TODO: report busy instead.
```

Comment:
Why don't fix it by using this PR's approach?

Reply:
They are not the same. Cases fixed above are raftstore errors, here is just the schedule error.

Reply:
Ok